crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2381,6 +2396,36 @@ impl From<&SymCacheError> for ObjectFileStatus {
     }
 }
 
+/// Merges the Stack Traces processed via Breakpad with the ones captured on the Client.
+///
+/// For now, this means we will prefer the client-side stack trace over the processed one, but in
+/// the future we could be a bit smarter about what to do.
+fn merge_clientside_with_processed_stacktraces(
+    processed_stacktraces: &mut [RawStacktrace],
+    clientside_stacktraces: Vec<RawStacktrace>,
```

Comment:
I wonder if a move here is more expensive than a reference?  Or would the compiler figure that out itself and do something smart?

Reply:
Hum yes.  So if this is huge we should allocate it on the heap?  I'm exploring the theoretical ideas behind this btw, not saying that this is a concern for us in this situation but rather seeing how well I understand rust ;)

Reply:
`Vec`s (and `HashMap`s) are always on the heap (well, the actual data is). Whats on the stack are just 3 usize: `ptr, len, capacity`.

Reply:
doh, yes of course.  bad brain.